frontend/tests/controllers-test/challengeHostTeamsCtrl.test.js

```diff
@@ -429,6 +429,31 @@ describe('Unit tests for challenge host team controller', function () {
         });
     });
 
+    describe('Unit tests for confirmMemberDelete function', function () {
+        beforeEach(function () {
+            spyOn($mdDialog, 'show').and.callFake(function () {
+                var deferred = $injector.get('$q').defer();
+                return deferred.promise;
+            });
+        });
+
+        it('open dialog to confirm delete', function () {
+            var hostTeamId = 1;
+            var hostId = 1;
+            var ev = new Event('$click');
+            var confirm = $mdDialog.confirm()
+                .title('Would you like to remove this member?')
+                .textContent('Note: This action will remove this member from the team.')

```

Comment:
change this `Note: This action will remove this member from the team.` to `Note: This action will remove the member from team.`